projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -1393,7 +1578,7 @@ private boolean processCommand(String[] words, FileWriter outWriter) {
          case LIST_TESTRIGS:
             return listTestrigs();
          case LOAD_QUESTIONS:
-            return loadQuestions(outWriter, parameters);
+            return loadQuestions(outWriter, parameters, _bfq);
```

Comment:
Was this change made to eventually support different question dictionaries for different containers? Or something else?